treeherder/model/management/commands/cycle_data.py

```diff
@@ -531,6 +578,17 @@ def add_arguments(self, parser):
             default=False,
             help='Write debug messages to stdout',
         )
+        parser.add_argument(
+            '--days',
+            action='store',
+            dest='days',
+            type=int,
+            help=(
+                f'Data cycle interval expressed in days. '
+                f'Minimum {MINIMUM_PERFHERDER_EXPIRE_INTERVAL} days '
+                f'when expiring performance data.'  # unless on prototype environment
```

Comment:
Do you mean "unless on prototype2 environment"?

Reply:
Yes, you are right. I'm reformulating this, to avoid confusion.